lib/motions/general-motions.coffee

```diff
@@ -107,6 +107,15 @@ class Motion
   isInclusive: ->
     @vimState.mode is 'visual' or @operatesInclusively
 
+  isBeginningOfFile: (cursor) ->
+    cur = cursor.getBufferPosition()
+    not cur.row and not cur.column
```

Comment:
I don't think `Point.equals` is defined and `is` doesn't work because the values are stored in `.row` and `.column`.